public/document-table-vis.js

```diff
@@ -18,104 +18,113 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { Schemas } from '../../../src/legacy/core_plugins/vis_default_editor/public';
-import { prepareJson, prepareString } from '../../../src/legacy/core_plugins/visualizations/public/np_ready/public/legacy/build_pipeline';
+import { Schemas } from '../../../src/plugins/vis_default_editor/public';
+import { prepareJson, prepareString } from '../../../src/plugins/visualizations/public/legacy/build_pipeline';
 
 import tableVisTemplate from './enhanced-table-vis.html';
-import { EnhancedTableVisualizationController } from './vis_controller';
+import { getEnhancedTableVisualizationController } from './vis_controller';
 import { enhancedTableRequestHandler } from './data_load/enhanced-table-request-handler';
 import { documentTableResponseHandler } from './data_load/document-table-response-handler';
 import { DocumentTableData } from './components/document_table_vis_data';
 import { EnhancedTableOptions } from './components/enhanced_table_vis_options';
 
+function toExpression(vis) {
+  const visConfig = { ...vis.params };
+  const { indexPattern, aggs } = vis.data;
+  let pipeline = `kibana | enhanced_table_visualization type='${vis.type.name}'
```

Comment:
Why did you add `kibana |` in this line?
I just tested without this prefix, and it works great.
I prefer to have the minimum pipeline expression, and `kibana` item is already put in expression by `build_pipeline.ts`

Reply:
I think at one point I was under the impression that the pipeline variable gets overwritten instead of appended to, as that's how it came out in the compiled code, but now that I look at the actual .ts files I realize that's not the case. I'm removing those  and pushing the changes in a minute.